app.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => UI.addBookList(book));
+  }
+
+  static addBookList(book) {
+    const bookList = document.getElementById('book-list');
+
+    const content = document.createElement('div');
+    content.innerHTML = `
+    <div>${book.title}</div>
+    <div>${book.author}</div>
+    <button class="delete">Remove</button>
+    <hr>
+    `;
+
+    bookList.appendChild(content);
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.remove();
+    }
+  }
```

Comment:
According to the project requirements:
![image](https://user-images.githubusercontent.com/42799579/197651484-06ba0e1a-4d27-4369-aaa3-39ee8ce2139f.png)

Your project currently deletes the book correctly from the page after clicking on the 'Remove' button, well done 😄! Please make sure to also remove it from your book collection. This means that, after removing a book from your page, it shouldn't come back after loading it again (you should update your localStorage 😉).

Your booklist:
![image](https://user-images.githubusercontent.com/42799579/197651738-5853ea78-e4de-41ba-b8ff-59bf945a1d36.png)
Your collection: 
![image](https://user-images.githubusercontent.com/42799579/197651769-d763aef6-8c5e-4e82-bc41-91809dfa5d71.png)

- - - - -

Booklist after removing the elements:
![image](https://user-images.githubusercontent.com/42799579/197651801-40064144-ea54-4ee1-b578-c48ce6e40eea.png)
Notice that the collection remains the same:
![image](https://user-images.githubusercontent.com/42799579/197651844-9ad4aa8f-bacb-4427-a339-9cdecbde9728.png)
And after refreshing the page, they appear again:
![image](https://user-images.githubusercontent.com/42799579/197651892-e4d9b56a-511a-432e-a1e3-0ab524996b0d.png)

> My tip for you: you can add ids to help you identify each book and use the [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method to remove a book, given an id 😃. Good luck and keep coding!